docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,91 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 Collect statistics of the snapshot with id `snap1` of table `my_table` for columns `col1` and `col2`
 ```sql
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
-```
\ No newline at end of file
+```
+
+## Table Replication
+
+The `rewrite-table-path` assists in moving or copying an Iceberg table from one location to another.
+
+### `rewrite-table-path`
+
+This procedure writes a new copy of the Iceberg table's metadata files where every path has had its prefix replaced.
+The newly rewritten metadata files, along with data files, enable moving or coping an Iceberg table to a new location.
```

Comment:
> The newly rewritten metadata files, along with data files, enable moving or coping an Iceberg table to a new location.

Do we need to call out the position delete rewrite as it can have old / sources files reference too ? to the best of my knowledge we did handle this case as if they will be re:written .

Reply:
Yeah, good point. I believe we rewrite not just metadata files, but also position delete which have path reference.